src/Catty/Views/Custom/CollectionViewCells/BrickCells/Pen/PenDownBrickCell.swift

```diff
@@ -0,0 +1,41 @@
+/**
+ *  Copyright (C) 2010-2020 The Catrobat Team
+ *  (http://developer.catrobat.org/credits)
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  An additional term exception under section 7 of the GNU Affero
+ *  General Public License, version 3, is available at
+ *  (http://developer.catrobat.org/license_additional_term)
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see http://www.gnu.org/licenses/.
+ */
+
+class PenDownBrickCell: BrickCell, BrickCellProtocol {
+
+    var textLabel = UILabel()
```

Comment:
Please do not initialize a new `UILabel` instance even though this instance is never used. Is there a reason why you do not use an optional here?

Reply:
There's no reason but I misunderstood that you wanted me to just remove implicit force unwrap. I will make it optional and update soon. :)

Reply:
Done!